simple_history/models.py

```diff
@@ -899,10 +900,14 @@ def related_manager_cls(self):
 
         class HistoricRelationModelManager(related_model._default_manager.__class__):
             def get_queryset(self):
+                cache_name = (
+                    # DEV: Remove this when support for Django 5.0 has been dropped
+                    self.field.remote_field.get_cache_name()
+                    if django.VERSION < (5, 1)
+                    else self.field.remote_field.cache_name
+                )
                 try:
-                    return self.instance._prefetched_objects_cache[
-                        self.field.remote_field.get_cache_name()
-                    ]
+                    return self.instance._prefetched_objects_cache[cache_name]
```

Comment:
Looks like [the `cache_name` property was added in Django 5.1](https://github.com/django/django/commit/b9838c65ec2fee36c1fd0d46494ba44da27a9b34), so I corrected the version numbers 🙂

Also, the tests didn't fail because of the `except (AttributeError, KeyError):` below, so I moved the `cache_name` variable out of the `try` block, to make the `except` only catch errors with the `_prefetched_objects_cache[cache_name]` line (`self.field.remote_field` should always have a `cache_name`/`get_cache_name()` value, to my understanding).